spec/rubocop/cop/style/round_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::Round do
+  subject(:cop) { described_class.new }
+
+  context 'with round' do
+    it 'registers an offense for an invocation with args' do
+      inspect_source(cop, '6.75.round(2)')
+      expect(cop.offenses.size).to eq(1)
+    end
+
+    it 'registers an offense for an invocation without args' do
+      inspect_source(cop, '6.75.round')
+      expect(cop.offenses.size).to eq(1)
+    end
+  end
+
+  context 'with ceil' do
+    after(:each) { expect(cop.offenses).to be_empty }
+
+    it 'does not register an offense for an invocation with args' do
+      inspect_source(cop, '6.75.ceil')
```

Comment:
This is not an invocation with args. It is the same source like the "without" spec.